src/simple/precise-code-intel/package.dhall

```diff
@@ -0,0 +1,45 @@
+let util = ../../util/package.dhall
+
+let Image = util.Image
+
+let Container = util.Container
+
+let image =
```

Comment:
See https://github.com/sourcegraph/deploy-sourcegraph-core/blob/f70cebe1813012b2f028bbd41c2379f6a98d577a/src/simple/indexed-search/package.dhall#L7

Reply:
Doing this way makes it trivial to update the targeted commit (versus needing to manually tweak the tag and digest)